db/seeds.rb

```diff
@@ -6,4 +6,3 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-Seeds::UserSeeder.new.run
```

Comment:
`Seeds` was not defined when running `db:seed` so I removed this and plan on just entering my seed commands directly in this file.